min_azimuth: 0.0
max_azimuth: 0.0
min_zenith: 0.0
max_zenith: 0.0
min_roll: 0.0
max_roll: 0.0
linear_step: 0.01
angular_step: 0.5
min_distance_to_object: -0.03
max_distance_to_object: -0.03
max_samples: 20
max_height: 0.0
min_height: 0.0
height_tolerance: 0.24
angular_tolerance: 10.0
target_frame: base_link
reference_axis: z
rotation_offset: 0.0
linear_offset_x: 0.0
linear_offset_y: 0.0
linear_offset_z: 0.0
generate_pregrasp_waypoint: True
joint_1_offset: 0.0
joint_2_offset: -0.6
joint_3_offset: 0.3
joint_4_offset: 0.0
joint_5_offset: 0.0
rotation_range_min: 270.0
rotation_range_max: 90.0
loop_rate: 10.0
move_group: move_group
arm: arm_1
gripper_config_matrix: youbot_dynamixel_gripper

